app/RulesPage/page.jsx

```diff
@@ -0,0 +1,15 @@
+// import Container from '@/components/Container'
+import RulesSections from '@/components/RulesSections'
+import sectionsData from '@/data/RulesPage/rulesData'
+import React from 'react'
+
+const RulesPage = () => {
+  return (
+    <div>
+      {/* <Container /> */}
+      <RulesSections sections={sectionsData.sections}/>
```

Comment:
`sectionsData` may be null since it comes from a dynamic and changeable file, `data.json`. Please ensure that `sectionsData` is not null and provide a fallback for such cases.